vulture_os/toolkit/api_parser/carbon_black/carbon_black.py

```diff
@@ -127,13 +127,9 @@ def get_logs(self, cursor=0, since=None):
             since += "Z"
 
         query = {
-            'criteria': {
-                'category': ['THREAT'],
-                'group_results': True,
-                'last_update_time': {
-                    'start': since,
-                    'end': datetime.now().strftime("%Y-%m-%dT%H:%M:%S.000Z")
-                }
+            'time_range': {
+                'start': since,
+                'end': datetime.now().strftime("%Y-%m-%dT%H:%M:%S.000Z")
```

Comment:
Do these parameters really result in the same query as before?
- are only the THREAT alert category fetched?
- does `time_range` act on the `last_update_time`?
- are logs still grouped?